### DISCUSSION TITLE
2022/02/07/Screeps%E5%85%A5%E9%97%A8%E6%95%99%E7%A8%8B2-%E5%86%9C%E6%B0%91/

### DISCUSSION THREAD

**Discussion by giscus** [Westlifers/Westlifers.github.io#13] [Announcements]
# 2022/02/07/Screeps%E5%85%A5%E9%97%A8%E6%95%99%E7%A8%8B2-%E5%86%9C%E6%B0%91/

本文将介绍基本的兵种分类，能量的来源、用处，最终学会如何生产一个智能农民进行能量采集。 兵种前文提到，爬根据身体部件的不同，分为多个兵种，但实际上官方并没有给出固定的兵种，其划分是取决于个人的。但在这里我们仍然要提到一些最基本且被广泛接受的一些兵种。本文涉及到：农民（harvester） 能量与spawn在正式开始编写代码之前，我们要先了解spawn与能量 能量每个房间有一些黄色的闪烁的点，这叫做

http://localhost:4000/2022/02/07/Screeps%E5%85%A5%E9%97%A8%E6%95%99%E7%A8%8B2-%E5%86%9C%E6%B0%91/

**Comment by Westlifers** [comment]
可以，深度好文！

**Comment by ghost** [comment]
收录详尽且通俗易懂，对新手友好度高！
